test/verify/check-openshift

```diff
@@ -398,6 +398,8 @@ LABEL io.projectatomic.nulecule.atomicappversion="0.1.11" \
         wait(lambda: "Running" in o.execute("oc get pods | grep virt-controller"), delay=2) # is it deployed and up?
         wait(lambda: "No resources found" in o.execute("oc get vm 2>&1"), delay=2) # No VM created yet
 
+        self.allow_journal_messages('.*type=1400.*')
```

Comment:
Please don't do this blanket exception of all SELinux violations. Which ones did you see? They usually should be reported downstream and then filtered out with naughty issues, or if they are that pervasive, at least get a comment and a stricter pattern.